base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py

```diff
@@ -12,11 +12,18 @@
 table = widget.getTable()
 row = 1
 color = ColorFontUtil.getColorFromRGB(240,245,245)
+
 i = 0
 for i in range(20):
-    reading = PVUtil.getString(pvs[i]) 
+    reading = PVUtil.getString(pvs[i])
+    temperature = PVUtil.getString(pvs[i+20])
+    drift = PVUtil.getString(pvs[i+40])
+     
     table.setCellText(row, 3, reading)
+    table.setCellText(row, 1, temperature)
+    table.setCellText(row, 2, drift)
```

Comment:
Although these numbers aren't repeated, it's good practice to have variables (preferably `static` and  `final` if we were in Java land)  `reading_column_index` etc. It makes future changes easier and maintains clarity if those variables end up being reused elsewhere.